### ISSUE TITLE
Syntax highlighting not working

### ISSUE THREAD

**Issue by webative-net** [gitlabhq/gitlabhq#10043]
in markdown I see some red lines, for code snippets, but in rest it doesn't work. I saw on js files, there is no highlighting at all. Also for yml files, they are like plain text. Do I have to install something for this to work, since I installed gitlab from source.

**Comment by stanhu** [comment]
There shouldn't be anything special to install; all the syntax highlighting is done on the backend. Did your filename include the right extension? (e.g. `test.js`) I'd suggest installing GitLab via the omnibus version to save yourself some hassle.

**Comment by webative-net** [comment]
@stanhu Thanks for your suggestion. Unfortunatelly the machine where I installed gitlab, has an ARM processor with linux 3.4 version, which does not support docker. Hence the manual install.

I noticed that highlighting works in snippets, w or w/o extensions. I think it detects the header of the file. For example I have a few shell scripts, without extension, which are properly highlighted when I view them in Snippets.

I will mark this as resolved for now.
